src/foam/u2/dialog/NotificationMessage.js

```diff
@@ -145,7 +145,7 @@ foam.CLASS({
         // Create notification message and description from
         // exception name and message.
         var ex = this.err.exception || this.err;
-        if ( ex.id ) {
+        if ( ex.id && ! this.message ) {
```

Comment:
Why are you doing this?  
We depend on the Exception name being the Title and message being the notification text.

Reply:
Checking for `! this.message` because it is the override error message from the client call. Without this the error message will be the exception class name eg. for ValidationException the message will be "Validation" and the actual error message will be in the description of the notification.

<img width="1600" alt="Screen Shot 2022-12-05 at 3 52 16 PM" src="https://user-images.githubusercontent.com/441658/205741074-3f4993ed-0ce3-4436-b858-ab4b04332bc0.png">

Reply:
Well then you need to design things so a generic ClientRuntimeException (or similar) sets things up like you have and other exceptions continue to work as already designed.